src/js/background.js

```diff
@@ -766,6 +789,51 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  showCookies: function(fqdn) {
+    let domain = window.getBaseDomain(fqdn);
+    chrome.cookies.getAll({domain}, cookies => {
+      console.log('Cookies for domain: ' + domain);
+      cookies.forEach(cookie => {
+        console.log('\t -------------');
+        console.log('\t cookie name: ' + cookie.name);
+        console.log('\t cookie domain: ' + cookie.domain);
+      });
+    });
+  },
+
+  debugSite: function() {
+    let self = this;
+    chrome.tabs.query({'active': true}, tabs => {
+      tabs.forEach(tab => {
```

Comment:
Could rescue two indentation levels (and make the code a bit easier to follow) by moving the anonymous function here into a private function inside of `debugSite` and then providing that function to `tabs.forEach`:
```javascript
  debugSite: function() {
     let self = this;

     function _debug(tab) {
       let action_map = ...
       ...
     }

    chrome.tabs.query({'active': true}, tabs => tabs.forEach(_debug));
  },
```